src/urllib3/util/ssl_.py

```diff
@@ -127,7 +157,7 @@ def assert_fingerprint(cert: Optional[bytes], fingerprint: str) -> None:
 
     if not hmac.compare_digest(cert_digest, fingerprint_bytes):
         raise SSLError(
-            f'Fingerprints did not match. Expected "{fingerprint}", got "{hexlify(cert_digest)}".'
+            f'Fingerprints did not match. Expected "{fingerprint}", got "{hexlify(cert_digest)!r}".'
```

Comment:
`!r` isn't what we want here, we'd receive a message that says `... got "'abcdef'"` (notice the `"` followed by `'`)

Reply:
Changed to use `bytes.hex()` (introduced in Python 3.5)